src/entity/Country.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, Column, PrimaryColumn, Index } from "typeorm";
+
+export enum Region {
+    AFRICA = "africa",
+    ASIA = "asia",
+    ANTARCTICA = "antarctica",
+    AUSTRALASIA = "australasia",
+    CARIBBEAN = "caribbean",
+    EUROPE = "europe",
+    MIDDLE_EAST = "middle east",
+    NORTH_AMERICA = "north america",
+    SOUTH_AMERICA = "south america",
+}
+
+@Entity()
+export class Country {
+    @PrimaryColumn({ type: "int2" })
+    id!: number;
+
+    @Column({ type: "varchar", length: 100 })
+    name!: string;
+
+    @Column({ type: "char", length: 2 })
+    code!: string;
+
+    @Column({ type: "float8" })
+    latitude!: number;
+
+    @Column({ type: "float8" })
+    longitude!: number;
```

Comment:
do we want to maybe also save coastline? we dont do anything with it yet, but its technically part of the static data and maybe if we ever figure out the naval effectiveness bands...

Reply:
good point. It isn't affected by nukes, right?

Reply:
uh i actually have no idea, i never considered it